pkg/client/session.go

```diff
@@ -0,0 +1,91 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"github.com/codenotary/immudb/pkg/api/schema"
+	"github.com/codenotary/immudb/pkg/client/cache"
+	"github.com/codenotary/immudb/pkg/client/errors"
+	"github.com/codenotary/immudb/pkg/client/heartbeater"
+	"github.com/codenotary/immudb/pkg/client/state"
+	"github.com/codenotary/immudb/pkg/signer"
+	"github.com/codenotary/immudb/pkg/stream"
+	"github.com/golang/protobuf/ptypes/empty"
+	"google.golang.org/grpc"
+)
+
+func (c *immuClient) OpenSession(ctx context.Context, user []byte, pass []byte, database string) (serverUUID string, sessionID string, err error) {
```

Comment:
I think there must not be an already opened session

Reply:
Why? It's possible to open many sessions as we want

Reply:
Yes, it's possible to create 1 session for each client